portal-web/test/functional/com/liferay/portalweb/macros/FormRules.macro

```diff
@@ -355,6 +384,35 @@ definition {
 			value1 = "${conditionOperatorLowerCase}");
 	}
 
+	macro viewConditionPresent {
+		var key_conditionRowNumber = "${conditionRowNumber}";
+		var key_conditionField = "${conditionField}";
+		var key_conditionOperator = "${conditionOperator}";
+
+		if (isSet(conditionValueType) && isSet(conditionFieldValue)) {
+			var key_conditionValueType = "${conditionValueType}";
+			var key_conditionFieldValue = "${conditionFieldValue}";
+
+			AssertElementPresent(locator1 = "FormRules#RULE_BUILDER_CONDITION_FIELD_PARAMETERS_EXTENDED");
+		}
+		else {
+			AssertElementPresent(locator1 = "FormRules#RULE_BUILDER_CONDITION_FIELD_PARAMETERS");
+		}
+	}
+
+	macro viewConditionSummaryNotPresent {
+		var key_ruleNumber = "${ruleNumber}";
+		var key_conditionField = "${conditionField}";
+		var key_conditionOperator = "${conditionOperator}";
+
+		if (isSet(conditionFieldConstant)) {
+			AssertElementNotPresent(locator1 = "FormRules#RULE_BUILDER_SUMMARY_CONDITION_EXTENDED");
+		}
```

Comment:
@lima-rafa, apparently, this is not necessary for this test, right? If so, can you please remove this code and the path `FormRules#RULE_BUILDER_SUMMARY_CONDITION_EXTENDED`?